ui/app/com/gu/recipeasy/controllers/Application.scala

```diff
@@ -133,6 +144,10 @@ class Application(override val wsClient: WSClient, override val conf: Configurat
       db.deleteCuratedRecipeByRecipeId(recipeId)
       db.insertCuratedRecipe(curatedRecipeWithId)
       db.moveStatusForward(recipeId)
+      db.getOriginalRecipeStatus(recipeId) match {
```

Comment:
You are 100% right, actually, but in this case I prefer `foreach`, as in 
```
db.getOriginalRecipeStatus(recipeId).foreach(status => db.insertUserEvent(UserEvent(request.user.email, request.user.firstName, request.user.lastName, recipeId, recipeStatusToUserEventOperationType(status))))
```
Corrected :)